streams/src/test/java/org/apache/kafka/streams/integration/KStreamAggregationIntegrationTest.java

```diff
@@ -43,10 +43,6 @@
 import org.junit.Before;
 import org.junit.ClassRule;
 import org.junit.Test;
-import org.junit.runner.RunWith;
```

Comment:
Because KStreamAggregationIntegrationTest was never supposed to run with caching, we have the other test KStreamAggregationDedupIntegrationTest that was supposed to contain the same tests but with caching.